Sources/Views/MessageLabel.swift

```diff
@@ -479,7 +479,7 @@ open class MessageLabel: UILabel {
             guard let date = date else { return }
             handleDate(date)
         case let .link(url):
-            guard let url = url, UIApplication.shared.canOpenURL(url) else { return }
+            guard let url = url, ["http", "https"].contains(url.scheme?.lowercased() ?? "") else { return }
```

Comment:
How about this?

```suggestion
            guard let scheme = url?.scheme?.lowercased(), ["http", "https"].contains(scheme) else { return }
```

Reply:
I think I would still need `let url = url` because url is optional, then it will be like this?

`
guard let url = url, let scheme = url?.scheme?.lowercased(), ["http", "https"].contains(scheme) else { return }
`

Reply:
I think you don't need `let url = url`. 
Because `scheme` always will be `nil` when `url` is `nil`.

Reply:
aw I mean I need to unwrap `url` to use -> `private func handleURL(_ url: URL)`

Reply:
ah~~, you right!
`guard let url = url, let scheme = url.scheme?.lowercased(), ["http", "https"].contains(scheme) else { return }`

Reply:
I changed my suggestion 👍

Reply:
Fixed!